app/views/content/non-uk-teachers/teach-in-england-if-you-trained-overseas.md

```diff
@@ -85,15 +85,52 @@ You may also be eligible for another type of visa, or you may have a pre-existin
 
 [If you're an Irish citizen you do not need a visa](https://www.gov.uk/government/publications/common-travel-area-guidance).
 
-## Consider getting qualified teacher status
+## Consider getting qualified teacher status (QTS)
 
-[You need 'qualified teacher status' (QTS) to teach in many schools in England](https://www.gov.uk/government/publications/qualified-teacher-status-routes-to-qts-for-teachers-and-those-with-teaching-experience-outside-the-uk).
+[Qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts) is the professional status teachers in England gain at the end of their teacher training.
 
-It's not a requirement in all [types of schools](https://www.gov.uk/types-of-school), such as academy schools, free schools, private schools and independent schools. However, having QTS can help your application.
+If you’re a qualified teacher from outside the UK, you can [work as a teacher in England for up to 4 years without QTS](https://www.gov.uk/government/publications/apply-for-qualified-teacher-status-qts-if-you-teach-outside-the-uk/routes-to-qualified-teacher-status-qts-for-teachers-and-those-with-teaching-experience-outside-the-uk#qts-exemption-for-teachers-from-outside-the-uk).
 
-If you’re a qualified teacher from outside the UK you can [work as a teacher in England for up to 4 years without QTS](https://www.gov.uk/guidance/recruit-teachers-from-overseas#employing-overseas-teachers-without-qts-the-4-year-rule).
+After that, you will need QTS to teach in many schools in England.
 
-[Apply for QTS if you trained outside the UK](https://www.gov.uk/government/publications/qualified-teacher-status-routes-to-qts-for-teachers-and-those-with-teaching-experience-outside-the-uk).
+It’s not a legal requirement in all types of schools (for example, [academy schools, free schools and private schools](https://www.gov.uk/types-of-school)).
+
+However, even in these schools, having QTS can help your application for a teaching job.
+
+### Understanding the different routes to QTS
```

Comment:
```suggestion
## The different routes to QTS
```
@laurahermionetennant92 do you think this might work better as a h2, and slightly shorter maybe?